packages/react/ds/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@govie-ds/react",
-  "version": "0.20.1",
+  "version": "0.20.2",
```

Comment:
Please don't update the version here. We have the release pipeline that will update it automatically.

Reply:
https://github.com/ogcio/govie-ds/commit/e970909744cf37e7c46af7bc794d27aaa1b7bfaf